headers/xdp/xdp_helpers.h

```diff
@@ -3,8 +3,17 @@
 #ifndef __XDP_HELPERS_H
 #define __XDP_HELPERS_H
 
+#define XDP_RUN_CONFIG_SECTION_ORIG "xdp_run_config"
+#define XDP_RUN_CONFIG_SECTION_NEW ".data.xdp_run_config"
+
+#if defined(USE_LIBBPF_V1_SECTION_NAMES)
```

Comment:
This define will be a user-facing API (as this header is meant to be included by user programs), so I guess we should document it somewhere. I'm not too fond of the name, but I can't think of anything better off the top of my head...